.misc/deploy.testpypi.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I've added what you suggested in the latest commit, it's just a test.
But isn't depending on environment variables a bit shakey?